csss-site/src/documents/models.py

```diff
@@ -99,22 +99,26 @@ class Media(models.Model):
     event = models.ForeignKey(
         Event,
         on_delete=models.CASCADE,
-        null=True
+        null=True,
+        blank=True
```

Comment:
why are you adding this?

Reply:
Crap... that was because I was adding events and media to the database for testing the photo gallery, and I added that so that I could add that object via the admin page while leaving the field for that attribute blank. I ran into some kind of deadlock while trying to add that object since I needed object A to add object B, but also needed object B to add object A

https://stackoverflow.com/questions/8609192/differentiate-null-true-blank-true-in-django

I can change that back if you want.

Reply:
fixed....